zanata-war/src/main/java/org/zanata/security/FacesSecurityEvents.java

```diff
@@ -34,6 +40,8 @@
  * Override the {@link org.jboss.seam.security.FacesSecurityEvents} component to
  * alter default values.
  *
+ * TODO [CDI] need to find a CDI replacement.
```

Comment:
Sorry my bad. It was extending the seam class and then I took it out. But it still uses seam classes in the implementation: 

import org.jboss.seam.international.StatusMessage; 
import org.jboss.seam.international.StatusMessages; 

It's still used as an event observer. I think it just puts some messages to jsf. @carlosmunoz may know better. 

----- Original Message -----

> From: "Sean Flanigan" <notifications@github.com>
> To: "zanata/zanata-server" <zanata-server@noreply.github.com>
> Cc: "Patrick Huang" <pahuang@redhat.com>
> Sent: Friday, July 31, 2015 9:43:38 AM
> Subject: Re: [zanata-server] Replace Seam Security (#908)

> In zanata-war/src/main/java/org/zanata/security/FacesSecurityEvents.java :
> > @@ -34,6 +40,8 @@
> >   * Override the {@link org.jboss.seam.security.FacesSecurityEvents}
> >   component to
> >   * alter default values.
> >   *
> > + * TODO [CDI] need to find a CDI replacement.
> On 2015-07-31 09:29, Patrick Huang wrote: In
> zanata-war/src/main/java/org/zanata/security/FacesSecurityEvents.java <
> https://github.com/zanata/zanata-server/pull/908#discussion_r35933312 >: >
> @@ -34,6 +40,8 @@ > * Override the { @link
> org.jboss.seam.security.FacesSecurityEvents} component to > * alter default
> values. > * > + * TODO [CDI] need to find a CDI replacement. This class
> extends org.jboss.seam.security.FacesSecurityEvents. Need to do it in
> deltaspike or something else.
> It doesn't extend anything in my checkout of this branch. Does this comment
> still apply? Actually, I think the class is entirely unused now.
> …
> -- Sean Flanigan Principal Software Engineer Globalisation Tools Engineering
> Red Hat

> —
> Reply to this email directly or view it on GitHub .